npc/instances/NydhoggsNest.txt

```diff
@@ -165,6 +165,9 @@ nyd_dun02,100,201,3	script	Yggdrasil Gatekeeper	HIDDEN_NPC,8,8,{
 				close;
 			goto L_Enter;
 		} else {
+			.@instance = has_instance2("1@nyd");
+			if (.@instance >= 0)
```

Comment:
Are you sure? there is an example on code style documentation using a condition without brackets inside another condition.

According to documentation brackets are needed when used "if, else if, else" and when if has more than one command, but when only has one command it doesn't put bracket, in exception when have an else an some side has more than two commands, in that case need brackets even if has only one command.

I check it twice by now, but if needed to change it I can change it inmediately.